src/photos/ducks/clustering/settings.js

```diff
@@ -0,0 +1,35 @@
+import { cozyClient, log } from 'cozy-konnector-libs'
```

Comment:
(this question is certainly addressed to @y-lohse :D) 

Why `cozy-konnector-libs` is exposing `cozyClient` ? Is it cozy-client or cozy-client-js ? We need it from there because `services` is executed in the same way as a connector ?

Reply:
Yes it is, the services are run in the same way as the connectors, so we use `cozy-konnector-libs` to have a `cozy-client-js` exposed and initialized.

Reply:
That being said, the only thing happening here with cozy-client-js is a fetch, which is definitely supported by `cozy-client` too. If we can find a good way to initialize cozy-client inside a service…

Reply:
Agreed,  if there is an easy way to use `cozy-client` on the service side, I'm up to use it